frontend/js/modules/dashboard.js

```diff
@@ -0,0 +1,457 @@
+module.exports = (function () {
+  'use strict';
+  // Global dependencies
+  var m  = require('mithril');
+  var ld = require('lodash');
+
+  // Local dependencies
+  var conf = require('../configuration.js');
+  var auth = require('../auth.js');
+  var u = auth.userInfo;
+  var layout = require('./layout.js');
+  var model = require('../model/group.js');
+  var sortingPreferences = require('../helpers/sortingPreferences.js');
+  var padWatch           = require('./pad-watch.js');
```

Comment:
Its better to be consistent with usage, no need to add extra whitespaces to align the = sign